flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/runtime/utils/BatchTestBase.scala

```diff
@@ -65,6 +66,12 @@ class BatchTestBase extends BatchAbstractTestBase {
   val LINE_COL_TWICE_PATTERN: Pattern = Pattern.compile("(?s)From line ([0-9]+),"
     + " column ([0-9]+) to line ([0-9]+), column ([0-9]+): (.*)")
 
+  /**
+    * Subclass should overwrite this method if we want to overwrite configuration during
+    * sql parse to sql to rel conversion phrase.
+    */
+  protected def getTableConfig: TableConfig = new TableConfig
```

Comment:
It is confused we expose the create a new TableConfig. Because we use the `tEnv.getConfig` to set a lot of configurations in `before()`. I think if we want to set sql parser configurations, we can also override `before()` method and add sql parse configs in `conf`.

Reply:
Can we add a TODO with the JIRA ID above this?  And tag Critical and 1.9 for the JIRA.

Reply:
Sure, i have fire a JIRA https://issues.apache.org/jira/browse/FLINK-13338: but without any `TODO` here.